### ISSUE TITLE
Getting issue when search query contains the ampersand character (&) 

### ISSUE THREAD

**Issue by Ljupch0** [ropensci/patentsview#24]
Hi, thanks for the useful package.

I have an issue when my query contains & when search for assignees.
```
result <- search_pv(
    query = with_qfuns(
        begins(assignee_organization = "Johnson & Johnson")
    ) ,
    fields = "assignee_organization",
    endpoint = "assignees",
    page = 1,
    per_page = 200
)
```
The error is: `Error: 'q' param is not valid json`

Simply removing it doesn't work as then it doesn't return the correct results.

**Comment by crew102** [comment]
Hi @Ljupch0 , thanks for bringing this to my attention. It's fixed in 2ea121e. If you don't want to bother installing the dev version to get the fix, you can just POST your request as such:

``` r
library(patentsview)

search_pv(
  query = with_qfuns(
    begins(assignee_organization = "Johnson & Johnson")
  ) ,
  fields = "assignee_organization",
  endpoint = "assignees",
  page = 1,
  per_page = 200,
  method = "POST"
)
#> $data
#> #### A list with a single data frame on an assignee level:
#> 
#> List of 1
#>  $ assignees:'data.frame':   34 obs. of  1 variable:
#>   ..$ assignee_organization: chr [1:34] "Johnson & Johnson Reserch Pty Limite"..
#> 
#> $query_results
#> #### Distinct entity counts across all downloadable pages of output:
#> 
#> total_assignee_count = 34
```

<sup>Created on 2021-06-17 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by Ljupch0** [comment]
Hi Chris, thanks for the quick fix! All the best.
